userguide/content/en/docs/Deployment/_index.md

```diff
@@ -33,7 +33,13 @@ Now that you're serving your site locally, Hugo will watch for changes to the co
 
 We recommend using [Netlify](https://www.netlify.com/) as a particularly simple way to serve your site from your Git provider (GitHub, GitLab, or BitBucket), with [continuous deployment](https://www.netlify.com/docs/continuous-deployment/), previews of the generated site when you or your users create pull requests against the doc repo, and more. Netlify is free to use for Open Source projects, with premium tiers if you require greater support.
 
-Follow the instructions in [Host on Netlify](https://gohugo.io/hosting-and-deployment/hosting-on-netlify/) to set up a Netlify account (if you don't have one already) and authorize access to your GitHub or other Git provider account. Once you're logged in:
+Before deploying with Netlify be sure to have set your repo as origin:
```

Comment:
I'm not sure if I'd phrase it quite like this as actually you need to have set the appropriate repo to origin *and* pushed to it, and it doesn't apply to users who have added Docsy using the UI or just as a theme.  We do document all these steps in the Getting Started section so how about instead:

```suggestion
Before deploying with Netlify, make sure that you've pushed your site source to your chosen GitHub (or other provider) repo, following any setup instructions in [Using the theme](/docs/getting-started/#using-the-theme).
```

...and then take out the command snippet. Agree it's good to remind people that Netlify won't work if you haven't got your repo set up! :)

Reply:
You are very right